docs/examples.md

```diff
@@ -122,8 +121,8 @@ bitcoin-cli scantxoutset start '[{"desc":"sh(wpkh(xpub6DP8WTA5cy2qWzdtjMUpLJHkzo
 {"xpub": "xpub6DP9afdc7qsz7s7mwAvciAR2dV6vPC3gyiQbqKDzDcPAq3UQChKPimHc3uCYfTTkpoXdwRTFnVTBdFpM9ysbf6KV34uMqkD3zXr6FzkJtcB"}
 ```
 
-2. With this xpub it is possible  extract the relevant UTXOs using the 
-`scantxoutset` in Bitcoin Core (from 0.17).
+2. With this xpub it is possible  extract the relevant UTXOs using the
+`scantxoutset` RPC call in Bitcoin Core.
```

Comment:
it's only in 0.17, still pertinent to tell users

Reply:
Am I understanding this correctly that you want it repeated on each of the 3 lines that mention `scantxoutset`? (as it used to be)

The purpose of the footnote was to reduce redundancy, but I can revert if you like.

Reply:
maybe it's ok, fine with as-is I guess